nfs-ganesha/build/build_deb

```diff
@@ -3,14 +3,21 @@ set -ex
 
 # Only do actual work when we are a DEB distro
 if test "$DISTRO" != "debian" -a "$DISTRO" != "ubuntu"; then
-    exit 0
+    exit 1
 fi
 
+REPO_URL="https://shaman.ceph.com/api/repos/ceph/$CEPH_BRANCH/$CEPH_SHA1/$DISTRO/$DIST/repo"
+if `curl --fail -L $REPO_URL`; then
+  echo "Ceph debian lib repo exists in shaman"
+else
+  echo "Ceph debian lib repo does NOT exist in shaman"
+  exit 1
+fi
 
 # We need this for system, to build ganesha, and to enable FSAL_CEPH and 
 # FSAL_RGW in the .spec file when cmake command runs
```

Comment:
This comment mentions "the .spec file", but this script is for building Debian packages. Want to update this?

It would also help to make it clearer in the comments exactly why we need this huge list of -dev packages when we're just going to use pbuilder to resolve them automatically. Does the `cmake` command or `make dist` command fail when those are not installed on the host?